diff --git a/server/presenters/env.js b/server/presenters/env.js
index adf16ac9..54e009c8 100644
--- a/server/presenters/env.js
+++ b/server/presenters/env.js
@@ -17,7 +17,7 @@ export default function present(env: Object): Object {
     SLACK_APP_ID: env.SLACK_APP_ID,
     MAXIMUM_IMPORT_SIZE: env.MAXIMUM_IMPORT_SIZE || 1024 * 1000 * 5,
     SUBDOMAINS_ENABLED: env.SUBDOMAINS_ENABLED === "true",
-    EMAIL_ENABLED: !!env.SMTP_USERNAME || env.NODE_ENV === "development",
+    EMAIL_ENABLED: !!env.SMTP_HOST || env.NODE_ENV === "development",
     GOOGLE_ANALYTICS_ID: env.GOOGLE_ANALYTICS_ID,
     RELEASE: env.SOURCE_COMMIT || env.SOURCE_VERSION || undefined,
   };
-- 
2.18.0