diff --git a/app/scenes/Settings/Features.tsx b/app/scenes/Settings/Features.tsx
index 4fe94b044..5fd14def3 100644
--- a/app/scenes/Settings/Features.tsx
+++ b/app/scenes/Settings/Features.tsx
@@ -11,6 +11,8 @@ import useCurrentTeam from "~/hooks/useCurrentTeam";
 import useStores from "~/hooks/useStores";
 import useToasts from "~/hooks/useToasts";
 import SettingRow from "./components/SettingRow";
+import Flex from "~/components/Flex";
+import Badge from "~/components/Badge";
 
 function Features() {
   const { auth } = useStores();
@@ -57,7 +59,7 @@ function Features() {
           />
         </SettingRow>
       )}
-      {/* <SettingRow
+      {<SettingRow
         name={TeamPreference.Commenting}
         label={
           <Flex align="center">
@@ -75,7 +77,7 @@ function Features() {
           disabled={!team.collaborativeEditing}
           onChange={handlePreferenceChange}
         />
-      </SettingRow> */}
+      </SettingRow>}
       {team.avatarUrl && (
         <SettingRow
           name={TeamPreference.PublicBranding}