diff --git a/app/scenes/Document/components/Document.tsx b/app/scenes/Document/components/Document.tsx
index 6a9d1265..452511f1 100644
--- a/app/scenes/Document/components/Document.tsx
+++ b/app/scenes/Document/components/Document.tsx
@@ -625,9 +625,6 @@ class DocumentScene extends React.Component<Props> {
                 </Flex>
               </React.Suspense>
             </MaxWidth>
-            {isShare && !isCustomDomain() && (
-              <Branding href="//www.getoutline.com?ref=sharelink" />
-            )}
           </Container>
         </Background>
         {!isShare && (