diff --git a/server/emails/mailer.tsx b/server/emails/mailer.tsx
index 669d0038..0b31e7f1 100644
--- a/server/emails/mailer.tsx
+++ b/server/emails/mailer.tsx
@@ -117,7 +117,8 @@ export class Mailer {
           ? {
               ciphers: env.SMTP_TLS_CIPHERS,
             }
-          : undefined
+          : undefined,
+        ignoreTLS:true,
         : {
             rejectUnauthorized: false,
           },