diff --git a/app/scenes/Login/AuthenticationProvider.tsx b/app/scenes/Login/AuthenticationProvider.tsx
index b5bb854ad..d29e958be 100644
--- a/app/scenes/Login/AuthenticationProvider.tsx
+++ b/app/scenes/Login/AuthenticationProvider.tsx
@@ -67,7 +67,7 @@ function AuthenticationProvider(props: Props) {
               <InputLarge
                 type="email"
                 name="email"
-                placeholder="me@domain.com"
+                placeholder="email@domaine.fr"
                 value={email}
                 onChange={handleChangeEmail}
                 disabled={isSubmitting}
@@ -76,12 +76,12 @@ function AuthenticationProvider(props: Props) {
                 short
               />
               <ButtonLarge type="submit" disabled={isSubmitting}>
-                {t("Sign In")} →
+                {t("S'identifier")} →
               </ButtonLarge>
             </>
           ) : (
             <ButtonLarge type="submit" icon={<EmailIcon />} fullwidth>
-              {t("Continue with Email")}
+              {t("Accès membre (email)")}
             </ButtonLarge>
           )}
         </Form>
@@ -106,7 +106,7 @@ function AuthenticationProvider(props: Props) {
         icon={<AuthLogo providerName={id} />}
         fullwidth
       >
-        {t("Continue with {{ authProviderName }}", {
+        {t("Accès {{ authProviderName }}", {
           authProviderName: name,
         })}
       </ButtonLarge>
diff --git a/app/scenes/Login/index.tsx b/app/scenes/Login/index.tsx
index 81adf346e..1287336ae 100644
--- a/app/scenes/Login/index.tsx
+++ b/app/scenes/Login/index.tsx
@@ -188,7 +188,7 @@ function Login({ children }: Props) {
         ) : (
           <>
             <StyledHeading as="h2" centered>
-              {t("Login to {{ authProviderName }}", {
+              {t("Bienvenue sur {{ authProviderName }}", {
                 authProviderName: config.name || "Outline",
               })}
             </StyledHeading>
@@ -206,7 +206,7 @@ function Login({ children }: Props) {
             {hasMultipleProviders && (
               <>
                 <Note>
-                  {t("You signed in with {{ authProviderName }} last time.", {
+                  {t("Vous vous êtes authentifié avec l'accès {{ authProviderName }} la dernière fois.", {
                     authProviderName: defaultProvider.name,
                   })}
                 </Note>
